modules/UI/invite/Invite.js

```diff
@@ -48,11 +50,6 @@ class Invite {
         APP.UI.addListener( UIEvents.INVITE_CLICKED,
                             () => { this.openLinkDialog(); });
 
-        APP.UI.addListener( UIEvents.INVITE_URL_INITIALISED,
```

Comment:
I think updateInviteUrl method is not used anymore. Maybe we should remove it too?